ai-service/tests/conftest.py

```
@@ -0,0 +1,32 @@
+from typing import Generator
```

Ruff (0.12.2)

Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)